clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -54,131 +46,73 @@ class ClioWebClient(
     }
   }
 
-  /**
```

Comment:
This all felt like overkill for the most common use-case, which is to spin up a web client, send 1 or 2 requests, then tear down.

Reply:
Hopefully a bit, but the main benefit is simplifying the code here.